ci/customization/customize_docs_in_folder.sh

```diff
@@ -46,7 +46,11 @@ for FILE in $(grep "${JTD_SEARCH_TERM}\|${DOXYGEN_SEARCH_TERM}\|${PYDATA_SEARCH_
   --exclude-dir=legacy \
   --exclude-dir=cudf-java \
   ${FOLDER_TO_CUSTOMIZE} ); do
-  python ${SCRIPT_SRC_FOLDER}/customize_doc.py $(realpath ${FILE})
+  if [[ ${FILE} == *"libucxx"* ]]; then
```

Comment:
I don't know what exactly this name matches against, but maybe we should change this to `*ucxx*` only? Note that we have three packages that we distribute:

1. [libucxx](https://anaconda.org/rapidsai-nightly/libucxx): C++ code;
2. [ucxx](https://anaconda.org/rapidsai-nightly/ucxx): Python/Cython code;
3. [distributed-ucxx](https://anaconda.org/rapidsai-nightly/distributed-ucxx): Python-only code (plugin for Dask/Distributed).

Reply:
Yeah that makes sense